corehq/apps/reports/templates/reports/bootstrap3/partials/base_maps.html

```diff
@@ -0,0 +1,142 @@
+{% load hq_shared_tags %}
+
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h2 class="panel-title">
+            {{ report.title }}
+        </h2>
+    </div>
+    <div class="panel-body">
+        <div class="hq-graphic-report">
+            {% block style %}
+            <link rel="stylesheet" href="{% new_static 'leaflet/dist/leaflet.css' %}" />
+            <link rel="stylesheet" href="{% new_static 'reports/css/maps.css' %}" />
+            {% endblock style %}
+            {% block js-libs %}
+            <script type="text/javascript" src="{% new_static 'jquery-color/jquery.color.js' %}"></script>
+            <script type="text/javascript" src="{% new_static 'leaflet/dist/leaflet.js' %}"></script>
+            {% endblock js-libs %}
+            {% block js-script %}
+            <script>
+
+            CONTEXT = {{ context|JSON }};
+            ICON_PATH = '{% static 'reports/css/leaflet/images' %}';
+            MIN_HEIGHT = 300; //px
+
+            $(document).ready(function() {
+                if (CONTEXT !== '') {
+                    load(CONTEXT, ICON_PATH);
+                }
+            });
+
+            </script>
+            {% endblock js-script %}
+
+            <script type="text/template" id="default_detail_popup">
+                <div class="default-popup">
+                    <h3><%= name %></h3>
+                    <hr>
+                    <table>
+                        <% _.each(info, function(field) { %>
+                        <tr class="data data-<%= field.slug %>">
+                            <td><%= field.label %></td>
+                            <td class="detail_data">
+                                <%= field.value %>
+                            </td>
+                        </tr>
+                        <% }); %>
+                    </table>
+                </div>
+            </script>
+
+            <script type="text/template" id="info_popup">
+                <% if (name) { %><h4><%= name %></h4><% } %>
+                <% _.each(info, function(field) { %>
+                <div><%= field.label %></div>
+                <div class="detail_data"><%= field.value %></div>
+                <% }); %>
+            </script>
+
+            <script type="text/template" id="legend_size">
+                <table class="sizelegend">
```

Comment:
Nevermind, looks like you're not inventing these names. Not a problem for this PR.